pkg/controller/infrastructure/templates/main.tpl.tf

```diff
@@ -84,6 +87,18 @@ resource "aws_route" "public" {
   }
 }
 
+{{ if and .enableDualstack .create.vpc }}
+resource "aws_route" "public-ipv6" {
+  route_table_id         = aws_route_table.routetable_main.id
+  destination_ipv6_cidr_block = "::/0"
+  gateway_id             = {{ .vpc.internetGatewayID }}
+
+  timeouts {
+    create = "5m"
```

Comment:
Does adding a route really take so long?
Why do we not specify a delete timeout?

Reply:
`timeouts` is copied from the route above. I wouldn't change it in the PR.